scrape/scrape_test.go

```diff
@@ -1331,7 +1331,7 @@ func TestScrapeLoopAppend(t *testing.T) {
 			// Honor Labels should ignore labels with the same name.
 			title:           "Honor Labels",
 			honorLabels:     true,
-			scrapeLabels:    `metric{n1="1" n2="2"} 0`,
+			scrapeLabels:    `metric{n1="1", n2="2"} 0`,
```

Comment:
Does the old way still work? As discussed, we should not change this behavior without a deeper discussion.

Reply:
Yes, the old way still does work... I guess I will leave in one example of this test, but I want to add a note that the syntax is invalid and may be removed if we decide it's a problem.